app/models/response.rb

```diff
@@ -4,4 +4,16 @@ class Response < ActiveRecord::Base
 
   validates :poll_id, presence: true
   validates :choice_id, presence: true
+
+  def question
+    self.choice.question.text
+  end
+
+  def text
+    self.choice.text
+  end
+
+  def question_choices
+    self.choice.question.choices
+  end
```

Comment:
If these are needed (see below), I would name the first two #question_text and #choice_text, to make it clear that they are returning an attribute and not an object (especially in the case of #question).

It is not clear why we would need these methods on the response model.  If this information is needed in relation to a poll, then it already available through poll.questions and poll.questions.choices.  Would it be more convenient to access this information there?